pepper-apis/dsm-core/src/main/resources/liquibase/rgp/PEPPER-833_rgp_kit_display_name.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="PEPPER-833_rgp_kit_type_display_name" author="pegah">
+        <addColumn tableName="kit_type">
+            <column name="display_name" type="VARCHAR(500)"></column>
```

Comment:
It looks like we are only setting this value for the new blood and rna type.  Is client code going to behave properly if this field is not null?  Should we put a unique key constraint on this new column and backfill it in some way that prefixes study guid or some such?

Reply:
yes. I changed the client code to first look at the display name and show that, if there is no display name then it should just work as it currently does and show the kit_type_name

Reply:
Can we encapsulate the "if display name is null, show the kit type name" into whatever dto we're using to supply both values so that client code does not have to do this check each wherever it's looking for the value?